README.md

```diff
@@ -136,4 +140,21 @@ Repeat this procedure for every custom action in the file:
 6. Copy the whole content of the `lke-configurations/plugins.json` file **except** for the `{` at the beginning and for the `}` at the end of the file
 7. On the `Plugin settings` field, add an new empty line before the last `}`, add in this line a comma (`,`) and, after the comma, paste the content copied in the step 6
 8. Click on `Save`
-9. Done!
\ No newline at end of file
+9. On Linkurious Enterprise dashboard, go to `Admin` -> `Data-source Schema`
+10. Scroll to the end of `NODES CATEGORIES & EDGES TYPES` list and click on `Create new categories`
+![](/assets/img/IMG_08.png)
+11. Type `REALESTATE_TRANSACTION` and click on `SAVE`
+12. Select the category you just created and, by clicking on `Create a new property`, create properties as following:
+    | Name             | Type   | Parameter                                    |
+    |------------------|--------|----------------------------------------------|
+    | address          | string |                                              |
+    | avg_sqft_price   | number |                                              |
+    | city             | string |                                              |
+    | contract_id      | string |                                              |
+    | price_range      | enum   | values: `OVERPRICED`, `UNDERPRICED`, `NORMAL`|
+    | purchase_price   | number |                                              |
```

Comment:
Now that we have the currency, we can better configure `purchase_price` so that people are at least aware of this. Even though today the data table plugin doesn't consider it